tests/test.py

```diff
@@ -67,6 +67,18 @@ def test_rename_file(self):
 
 		fs.unlink(new_filename)
 
+	def test_write_large_file(self):
+		test_file_path = '/largefile'
+		fs = Fatx('xbox_hdd.img')
+
+		b = b'1' * 1024 * 1024 # 1MB
```

Comment:
This is good but might fail to catch some offset/alignment issues because every cluster has exactly the same data. Let's use some sequence here so that it would fail in those cases

Reply:
Done, used 1MB of random bytes with a set seed

Reply:
Do you know what version of python the Ubuntu builder uses? I'm using `Random.randbytes` to generate the data, which requires python 3.9. I tried using `Random.randint` and casting to bytes as an alternative, but it is significantly slower (0.005 seconds with randbytes to 0.5 seconds with randint).

Reply:
I'd like to support Python 3.8, which Ubuntu 20.04 ships with

How about this? https://stackoverflow.com/a/28131868

Reply:
Done. It's still slower (0.063 seconds on my machine) but that's not too big of a deal.